portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsUploadField.testcase

```diff
@@ -153,13 +153,50 @@ definition {
 	}
 
 	@description = "Verify that a help text can be added to an Upload Field"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test HelpTextCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133221 HelpTextCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Upload");
+
+		FormViewBuilder.setFieldHelpText(fieldLabel = "Test Help Text");
+
+		FormViewBuilder.assertHelpTextPresent(
+			fieldName = "Upload",
+			fieldValue = "Test Help Text");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormViewBuilder.assertHelpTextPresent(
+			fieldName = "Upload",
+			fieldValue = "Test Help Text");
+
+		ItemSelector.gotoItemSelectorViaSelectButton();
+		
+		ItemSelector.uploadFile(
+			navTab = "Documents and Media",
+			uploadFileName = "Document_1.docx");
+
+		FormPortlet.submit();
```

Comment:
hi, @leilanyaragao , Please use FormPortlet.submitSuccessfullyAndViewSuccessPage instead of  FormPortlet.submit and viewSuccessPage